assignments/array-methods.js

```diff
@@ -56,21 +56,48 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
-console.log(fullName);
+
+runners.forEach(function(element){
+    fullName.push(element.first_name);
+    fullName.push(element.last_name);
+
+});
+
+console.log(fullName);  
+
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map(function(currentValue){
+    return allCaps.push(currentValue.first_name.toUpperCase());
+
+});
+
+
```

Comment:
Great work here! 
Here's the arrow function so you have it: 

```suggestion
let allCaps = runners.map(person => person.first_name.toUpperCase());

console.log(allCaps);
```